# PursuerEvaderDataArray.msg
Header header
PursuerEvaderData[] pursuer_evader_info_array
